sys/cbor/cbor.c

```diff
@@ -236,12 +233,12 @@ void dump_memory(const unsigned char *data, size_t size)
         return;
     }
 
-    DEBUG("0x");
+    printf("0x");
```

Comment:
It seems they are optimized out when you don't use them so I will remove the `CBOR_PRINT` module